### PR TITLE
Crazy up

### PR DESCRIPTION
**Pull request by ekg**
Improvement of wfmash's chaining model and high level wavefront reduction allow us to apply a smaller segment size. This improves sensitivity to inversions and larger structural variant breakpoints.

At the same time, we resolve a long-standing problem with smoothxg. We applied abPOA in global mode. This yields excellent runtime performance, but it can cause "accordions" in the graphs, which are characterized by runs of SNPs and indels. In effect, these are regions that are only aligned together because of our use of global alignment. Applying local alignment previously led to longer runtime (which was problematic) and also underaligned regions. By iterating smoothxg twice, we mitigate underalignment issues.

Basically, we were stuck in a suboptimal configuration using global alignment, but improvements to wfmash and the progressive application of smoothxg seem to have resolved these, allowing us to use local abPOA.

### CODE REVIEW

**Comment by AndreaGuarracino** [comment]
POA in local mode offers great benefits, but this mode seems not quite stable in abPOA. In the meantime, the latest changes to SPOA have improved its performance, so we give it a second chance to be the default tool for smoothing.
